test/common/network/udp_listener_impl_test.cc

```diff
@@ -397,6 +414,133 @@ TEST_P(UdpListenerImplTest, SendDataError) {
   EXPECT_DEATH(listener_->send(send_data), "Invalid argument passed in");
 }
 
+/**
+ * Test that multiple stacked packets of the same size are properly segmented
+ * when UDP GRO is enabled on the platform.
+ *
+ * [Only run the following test if UDP_GRO is configured]
```

Comment:
This is redundant with the line below, suggest removing.

Reply:
Done! Removed the redundant comment.